terragrunt/modules/crates-io/compute-static/src/main.rs

```diff
@@ -1,18 +1,22 @@
-use fastly::http::{Method, StatusCode};
+use fastly::http::{Method, StatusCode, Version};
 use fastly::{Error, Request, Response};
 use log::{info, warn, LevelFilter};
 use log_fastly::Logger;
 use once_cell::sync::Lazy;
 use regex::Regex;
 use serde_json::json;
+use std::env::var;
 use time::OffsetDateTime;
 
 use crate::config::Config;
-use crate::log_line::{LogLine, LogLineV1Builder};
+use crate::log_line::{HttpDetailsBuilder, LogLine, LogLineV1Builder, TlsDetailsBuilder};
 
 mod config;
 mod log_line;
 
+const DATADOG_APP: &str = "crates.io";
+const DATADOG_SERVICE: &str = "static.crates.io";
```

Comment:
Originally, these two static constants were defined in Terragrunt. But they don't change depending on the environment and hardcoding them removes a potential panic if they'd accidentally got deleted.